### DISCUSSION TITLE
What is your hosting environment?

### DISCUSSION THREAD

**Discussion by dbeavon** [dotnet/spark#1172] [General]
If you see this, can you please tell me what your primary hosting environment is for your .Net solution?

(you don't need to include on-prem standalone OSS).

The reason I'm asking is because I'm abandoning Synapse Analytics this year (isn't everyone) and I'm searching for a new place to run my C#.net workloads for Apache Spark.  I'm leaning towards HDInsight on Azure at the moment.  I had used Azure Databricks in the past but wasn't that successful (at least not on the cluster types and modes that were available to me at the time).  

I'm just wondering if HDInsight has become the most popular option at the moment.   It would be interesting to see what hosting platforms everyone is using (HDInsight or Databricks or  EMR or other ) 

Unfortunately Synapse Analytics doesn't have enough extensibility hooks to allow us to get .Net working.  It is sort of a shock that c# used to be a first class language (with notebook functionality and everything!)  But now it isn't possible to get c# working ... not even by if we wanted to introduce it ourselves.

**Comment by dbeavon** [comment]
I'll go first - HDI

**Reply by bolcman** [reply]
EMR, works pretty good so far.

**Comment by dbeavon** [comment]
I'm abandoning HDI.  
Back to Databricks.

Microsoft is slowly killing HDInsight, just like they did Synapse Analystics.

**Reply by GeorgeS2019** [reply]
Is this decision related to how Agentic AI taking over enterprise data flow processing??

**Reply by dbeavon** [reply]
No, I think Microsoft thinks they can make more money on Fabric.
Its only about their profit margins.  Low-code users will spend a lot more money.  Moreso than the types of folks who were using HDInsight.
